parser/parser_alter.go

```diff
@@ -530,6 +530,28 @@ func (p *Parser) parseAlterTableModify(pos Pos) (AlterTableExpr, error) {
 	switch {
 	case p.matchKeyword(KeywordColumn):
 		return p.parseAlterTableModifyColumn(pos)
+	case p.matchKeyword(KeywordTtl):
+		_ = p.lexer.consumeToken()
+		ttlExpr, err := p.parseTTLExpr(p.Pos())
+		if err != nil {
+			return nil, err
+		}
+		return &AlterTableModifyTTL{
+			ModifyPos:    pos,
+			StatementEnd: ttlExpr.End(),
+			TTL:          ttlExpr,
+		}, nil
+	case p.matchKeyword(KeywordRemove):
```

Comment:
This has been implemented here:
https://github.com/AfterShip/clickhouse-sql-parser/blob/e45d11299ca34a23ded2c6cd5eabfe6c52587707/parser/parser_alter.go#L42

And the syntax is: ALTER TABLE [db.]table_name [ON CLUSTER cluster] REMOVE TTL